ratom/metrics/license.go

```diff
@@ -72,8 +72,6 @@ func License(directory string) bool {
 	}
 	licenses := [9]string{"MIT", "LGPLv2.1", "Expat", "X11", "MPL-2.0", "Mozilla Public", "Artistic License 2", "GPLv2", "GPLv3"}
 
-	defer os.RemoveAll(directory)
```

Comment:
Original reason for this line and for removing it now?

Reply:
Looks like you just moved it to later in the program. If thats correct then all good